tests/e2e/edit-account.test.js

```diff
@@ -0,0 +1,191 @@
+  await understandButton.click();
+  await page.waitForTestIdSelector(`understand-${name}-button`, { hidden: true });
+};
+
+const openSelectionModalButtonClick = async (page) => {
+  const openSelectionModalButton = await page.getByTestId('open-selection-modal-button', true);
+  await openSelectionModalButton.click();
+};
+
+const selectICNSNameClick = async (page, icnsName) => {
+  const selectICNSName = await page.getByTestId(`icns-name-${icnsName}`, true);
+  await selectICNSName.click();
+};
+
+const viewAdditionalInfoButtonClick = async (page) => {
+  const viewAdditionalInfoButton = await page.getByTestId('view-more-button', true);
+  await viewAdditionalInfoButton.click();
+};
+
+const copyIdButtonClick = async (page, name, id) => {
+  const copyIdButton = await page.getByTestId(`copy-${name}-button`, true);
+  await copyIdButton.click();
+  const copiedText = await page.evaluate(() => navigator.clipboard.readText());
+
+  expect(copiedText).toBe(id);
+};
+
+const infoIconButtonClick = async (page, name) => {
+  const infoAccountIdButton = await page.getByTestId(`info-${name}-icon-button`, true);
+  await infoAccountIdButton.click();
+};
+
+const learnMoreButtonClick = async (browser, page) => {
+  const newPagePromise = new Promise((x) => browser.once('targetcreated', (target) => x(target.page())));
+  const learnMoreButton = await page.getByTestId('learn-more-button', true);
+  await learnMoreButton.click();
+  await page.waitForTimeout(3000);
+
+  const newPage = await newPagePromise;
+  const url = await newPage.evaluate(() => document.location.href);
+  expect(url).toBe('https://medium.com/plugwallet/internet-computer-ids-101-669b192a2ace');
+};
+
+const waitForWalletName = async (page) => {
+  await page.waitForTimeout(500);
+  await popupPageUtils.refreshWallet(page);
+
+  const name = await getWalletName(page);
+  const icnsName = 'bananas.icp';
+
+  if (name !== icnsName) {
+    return waitForWalletName(page);
+  }
+
+  return name;
+};
+
+// Utilities
+
+describe('Account Details View', () => {
+  let browser;
+  let page;
+  const principalId = 'principalId';
+  const accountId = 'accountId';
+  const icns = 'icns';
+
+  const icnsName = 'bananas.icp';
+  const AccountName = 'Account 122';
+
+  beforeAll(async () => {
+    browser = await setupChrome();
+
+    // Importing and unlocking the accoun
+    page = await utils.createNewPage(browser);
+
+    await optionsPageUtils.importAccount(page, secrets.seedphrase, secrets.password);
+    await optionsPageUtils.unlock(page, secrets.password);
+
+    await page.close();
+  });
+
+  beforeEach(async () => {
+    page = await utils.createNewPage(browser);
+    await page.goto(chromeData.popupUrl);
+    await popupPageUtils.profileButtonClick(page);
+  });
+
+  afterEach(async () => {
+    await page.close();
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  test('editing account name', async () => {
```

Comment:
This test is tricky because you are editing, what I would do is change the name to a new one, and then restore the name to the original one, so you don't need to print a new one each time the test is run, I know there are better ideas and more sophisticated ones, but this could help